modules/graph/vertex_map/arrow_vertex_map_impl.h

```diff
@@ -388,6 +429,136 @@ ObjectID ArrowVertexMap<OID_T, VID_T>::addNewVertexLabels(
   return ret;
 }
 
+template <typename OID_T, typename VID_T>
+ObjectID ArrowVertexMap<OID_T, VID_T>::updateLabelVertexMap(
+    Client& client, PropertyGraphSchema::LabelId label_id,
+    std::vector<std::vector<std::shared_ptr<oid_array_t>>>
+        oid_arrays){
+  using vineyard_oid_array_t =
+      typename InternalType<oid_t>::vineyard_array_type;
+  std::vector<vineyard_oid_array_t> vy_oid_array(fnum_);
+  std::vector<vineyard::Hashmap<oid_t, vid_t>> vy_o2g(fnum_);
+  int total_label_num = label_num_;
+  // every worker should update vm using their own oid_array
+  auto fn = [this,&label_id,&client,&oid_arrays,&vy_oid_array,&vy_o2g](const label_id_t label,const fid_t fid) -> Status{
```

Comment:
give `fn` a meaningful name, and document about its purpose, since it's such a complex lambda

Reply:
Yes, of course. Thanks a lot for your careful review.